paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -228,6 +238,66 @@ void CPUQuantizePass::QuantizeConv(Graph* graph,
   PrettyLogDetail(msg_ss.str().c_str());
 }
 
+void CPUQuantizePass::QuantizeFc(Graph* graph) const {
+  GraphPatternDetector gpd;
+  auto pattern = gpd.mutable_pattern();
+  patterns::FCMKLDNN fc_pattern{pattern, name_scope_};
+  auto* fc_input = gpd.mutable_pattern()
+                       ->NewNode("fc_quantizer/input")
+                       ->AsInput()
+                       ->assert_is_op_input("fc", "Input");
+  fc_pattern(fc_input, false);
+
+  int quantize_fc_count = 0;
+  auto handler = [&](const GraphPatternDetector::subgraph_t& subgraph,
+                     Graph* g) {
+    VLOG(4) << "Quantize fc op";
+    GET_IR_NODE_FROM_SUBGRAPH(fc, fc, fc_pattern);
+    auto* fc_op_desc = fc->Op();
+
+    // skip if should not be quantized
+    if (fc_op_desc->GetAttrIfExists<bool>("use_quantizer") != true ||
+        fc_op_desc->GetAttrIfExists<bool>("use_mkldnn") != true)
```

Comment:
Could you please provide my the link to the code that does that?

Reply:
As far as I recall, I have added it specifically here, since without this check when the topology was run without `fc_mkldnn_pass` it was actually quantizing the fp32 fc and then it was failing on finding a kernel that supports uint8 or int8 input.

Reply:
Yup, it's this commit a417bdb0de244febc72fd72c1526180ccb048fed that introduced that. Please see its title.

Reply:
True, thanks.